Docs/add-coadmins.md

```diff
@@ -0,0 +1,23 @@
+# Add co administrators on your Azure subscription
```

Comment:
Let's update the [Guidance on customizing preconfigured solutions doc](https://azure.microsoft.com/en-us/documentation/articles/iot-suite-guidance-on-customizing-preconfigured-solutions/#manage-the-permissions-in-a-preconfigured-solution) with steps below.